modules/extension/xsd/xsd-wfs/src/main/java/org/geotools/wfs/v1_1/PropertyTypeBinding_1_1.java

```diff
@@ -0,0 +1,103 @@
+import org.geotools.gml3.simple.GenericGeometryEncoder;
+import org.geotools.wfs.WFS;
+import org.geotools.xsd.AbstractComplexEMFBinding;
+import org.geotools.xsd.ElementInstance;
+import org.geotools.xsd.Encoder;
+import org.geotools.xsd.EncoderDelegate;
+import org.geotools.xsd.Node;
+import org.locationtech.jts.geom.Geometry;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.xml.sax.helpers.AttributesImpl;
+import org.xml.sax.helpers.NamespaceSupport;
+
+public class PropertyTypeBinding_1_1 extends AbstractComplexEMFBinding {
+
+    private static final String VALUE = "Value";
+
+    public PropertyTypeBinding_1_1(WfsFactory factory) {
+        super(factory);
+    }
+
+    @Override
+    public QName getTarget() {
+        return org.geotools.wfs.WFS.PropertyType;
+    }
+
+    @Override
+    public Element encode(Object object, Document document, Element value) throws Exception {
+        return value;
+    }
+
+    @Override
+    public Object parse(ElementInstance instance, Node node, Object value) throws Exception {
+        return super.parse(instance, node, value);
+    }
+
+    @Override
+    public Object getProperty(final Object object, QName name) throws Exception {
+        if (VALUE.equals(name.getLocalPart())) {
+            return (EncoderDelegate)
+                    output -> {
+                        Object value = ((PropertyType) object).getValue();
+
+                        output.startElement(
+                                org.geotools.wfs.WFS.NAMESPACE,
+                                VALUE,
+                                "wfs:" + VALUE,
+                                new AttributesImpl());
+                        if (value instanceof Geometry) {
+                            Geometry geometry = (Geometry) value;
+
+                            GMLConfiguration gml = new GMLConfiguration();
+
+                            Encoder encoder = new Encoder(gml);
+                            encoder.setInline(true);
+
+                            GenericGeometryEncoder geometryEncoder =
+                                    new GenericGeometryEncoder(encoder);
+                            GMLWriter handler =
+                                    new GMLWriter(
+                                            output,
+                                            new NamespaceSupport(),
+                                            gml.getNumDecimals(),
+                                            gml.getForceDecimalEncoding(),
+                                            gml.getPadWithZeros(),
+                                            "gml",
+                                            gml.getEncodeMeasures());
+                            geometryEncoder.encode(geometry, new AttributesImpl(), handler);
+
+                        } else {
+                            String s = value.toString();
+                            output.characters(s.toCharArray(), 0, s.length());
```

Comment:
I see you basically adapted the WFS 1.0 PropertyTypeBinding, so it should work, but this part is leaving me wondering.... are dates encoded as ISO timestamps when going through this path?
All other types I can think of should "toString" into a valid GML representation.

Not that it's necessarily a blocker either... what happens now? If java.util.Data is toStringed anyways, this change would not introduce a regression and would be good to go as-is

Reply:
I copied this code from the PropertyTypeBinding class and changed only the geometry transformation in it. 
I had to do this because WFS1.0 should be handled as before, and PropertyTypeBinding_1_1 should be used in WFS1.1. 
My goal was to fix the geometry handling and not change the rest of the code, I think this should be a new fix task.

Also, I think the problem will be not only with the date; if the user wants to set the field as null, an error will occur. 
But this is now beyond the scope of my correction. In the future, I will try to offer my own edits on this matter.

Reply:
You basically repeated what I said above... with one exception. Do you know what the current encoding output for date is, or not?

I've modified the WFSEncodingTest.encodeUpdate method to encode a Date, and it comes out as "Mon Dec 16 14:20:35 CET 2024"... it's not conformant. However, if I modify encodeInsert the same way (giving the encoder knowledge that I want a timestamp thought the attribute descriptor), then I get it encoded properly "2024-12-16T13:23:35.355Z"... it's however a different code path and has more context to judge the correct course of action.

I have no proof that the new code might introduce regressions in dates encoding, just a suspicion, so I'll let it go. Finger crossed.

Reply:
We did not pay attention to the date encoding; if I encounter problems regarding this, I will definitely offer my solution.